packages-content-model/roosterjs-content-model-plugins/lib/autoFormat/utils/convertAlphaToDecimals.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @internal
+ * Convert english alphabet numbers into decimal numbers
+ * @param letter The letter that needs to be converted
+ * @returns
+ */
+export default function convertAlphaToDecimals(letter: string): number | undefined {
```

Comment:
I think we already have these methods in `updateListMetadata.ts`, can we just reuse?

Reply:
We have `convertDecimalsToAlpha` that convert the number to the character, here we need to transform the character into number.